### PR TITLE
Carousel component

### LINKED ISSUE #38: Carousel components
Add carousel components to the stylesheet.

### PR DESCRIPTION
**Pull request by MariaCheca**
Closes #38

Carousel component:
screen shot 2016-02-05 at 10 44 37

CR @xavijam

### CODE REVIEW

File: src/scss/cdb-components/shapes/toogle-arrow.scss [hunk 1]
```diff
@@ -7,17 +7,17 @@ Description
 <h2>16px</h2>
 <div style="padding: 20px;">
   <div class="CDB-Shape" style="background: rgba(0, 0, 0, .02);">
-    <div class="CDB-ArrowToogle"></div>
+    <div class="CDB-ArrowToggle"></div>
```
Comment (xavijam) [inline src/scss/cdb-components/shapes/toogle-arrow.scss]: In my opinion, if we keep using shapes, we should name them as:

``` css
.CDB-Shape-arrowToggle {}
.CDB-Shape-magnify {}
```

cc @piensaenpixel @javierarce

File: src/scss/cdb-variables/_sizes.scss [hunk 2]
```diff
@@ -9,8 +9,8 @@ $mobileLayout: 280px;
 $baseSize: 8px;
 
 // Margins
-$sMargin-section: 24px; // To separate main section
-$sMargin-element: 14px; // To separate elements inside a group
+$sMargin-section: 24px;  // To separate main section
```
Comment (xavijam) [inline src/scss/cdb-variables/_sizes.scss]: Remove these spaces

File: src/scss/cdb-components/shapes/toogle-arrow.scss [hunk 3]
```diff
@@ -84,6 +84,7 @@ Description
 */
 
 @import '../../cdb-variables/colors';
+@import '../../cdb-variables/sizes';
```
Comment (xavijam) [inline src/scss/cdb-components/shapes/toogle-arrow.scss]: Why do you need this?

File: src/scss/cdb-utilities/helpers.scss [hunk 4]
```diff
@@ -2,7 +2,9 @@
 // ----------------------------------------------
 
 @import './mixins';
+@import '../cdb-variables/colors';
```
Comment (xavijam) [inline src/scss/cdb-utilities/helpers.scss:5]: Remove colors import because you don't need it

**Comment by xavijam** [comment]
Just some comments, for me :+1:

File: src/scss/cdb-components/carousel.scss [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+    border: 1px solid $cMainLine;
+    cursor: pointer;
+  }
+
+  &.is-selected {
+    border: 2px solid $cBlue;
+  }
+}
+
+.CDB-Carousel-headerDetails {
```
Comment (xavijam) [inline src/scss/cdb-components/carousel.scss]: Remove this thing because it is not needed.

**Comment by xavijam** [comment]
Remove fonts please.

**Comment by MariaCheca** [comment]
CartoIcon files restored :+1:

**Comment by piensaenpixel** [comment]
Only two things:
- I think vertical-align: top: is not necessary so all the elements have the same height
- I'm not sure about this one 
  &:not(:last-child) {
  margin-right: 8px;
  }

**Comment by piensaenpixel** [comment]
@MariaCheca one easy thing, could you prepare the component for the inside content could be center vertical and horizontal? :)

**Comment by MariaCheca** [comment]
@piensaenpixel done!

**Comment by piensaenpixel** [comment]
fix the conficts with master and for me its ok :dancer:

**Comment by MariaCheca** [comment]
:sparkles:

**Comment by xavijam** [comment]
:clap:

**Pull request merged**